Oracle/shrine_ACT_metadata_and_concept.sql

```diff
@@ -413,9 +413,9 @@ select /*+  PARALLEL (20) */ distinct
 from 
 (
 -- remove following table(HERON_TERMS) during normal heron release.
-select C_BASECODE, C_FULLNAME , C_NAME , UPDATE_DATE , DOWNLOAD_DATE,sourcesystem_cd, C_DIMCODE
-from BLUEHERONMETADATA.HERON_TERMS
-union all
+-- select C_BASECODE, C_FULLNAME , C_NAME , UPDATE_DATE , DOWNLOAD_DATE,sourcesystem_cd, C_DIMCODE
+-- from BLUEHERONMETADATA.HERON_TERMS
+-- union all
```

Comment:
If there's a reason to leave the code in commented out, please give that reason in a comment.

Otherwise, please delete the code. git will remember.